src/api/tests/test_admin.py

```
@@ -335,10 +335,11 @@ def test_admin_can_delete_spot():
 @pytest.mark.django_db
 def test_admin_can_assign_user_as_operator():
     """Admin can create operator profile for a user"""
-    User.objects.create_superuser(username="admin")
+    admin = User.objects.create_superuser(username="admin")
```

Ruff (0.14.3)

Local variable `admin` is assigned to but never used

Remove assignment to unused variable `admin`

(F841)